mgmt/LocalManager.cc

```diff
@@ -1025,6 +1049,19 @@ LocalManager::bindProxyPort(HttpProxyPort &port)
     }
   }
 
+  if (port.m_mptcp && mptcp_supported()) {
```

Comment:
I think it it would be better to not allow `m_mptcp` to be set if it is not supported. Please look at the transparency support code, which dealt with a similar issue back when transparency support require a special kernel mod. In essence, if the port descriptor has "mptcp" and it's not supported, issue a warning at that point and don't enable `m_mptcp`. The state of support isn't going to change while TS is running.

Reply:
Thanks for the feedback. I did it the way you described, checking for ```mptcp_supported``` in ```processOptions``` and based on that enable ```m_mptcp``` or not.

I just pushed the update to my branch here.